cli/src/lib/npm/npmLibDefs.js

```diff
@@ -408,8 +408,8 @@ export async function getInstalledNpmLibDefs(
               flowVerMatches == null
                 ? matches[3]
                 : flowVerMatches[3] == null
-                  ? flowVerMatches[2]
-                  : `${flowVerMatches[2]}-${flowVerMatches[4]}`;
+                ? flowVerMatches[2]
+                : `${flowVerMatches[2]}-${flowVerMatches[4]}`;
```

Comment:
interesting change from prettier O_o I really like the previous formatting more.

Reply:
This is just prettier hating nested ternaries